hack/verify-golint.sh

```diff
@@ -46,7 +41,7 @@ else
 fi
 
 if [[ -n "${bad_files}" ]]; then
-  echo "golint detected following problems:"
+  echo "!!! golint detected the following problems:"
```

Comment:
Would be better to change both to use `os::log::*` instead of this - we don't need twenty different ways to indicate an error.